src/h5web/vis-packs/core/line/hooks.ts

```diff
@@ -1,14 +1,14 @@
 import { useMemo } from 'react';
-import { useThree } from '@react-three/fiber';
 import { Vector3 } from 'three';
 import { useCanvasScales } from '../hooks';
 
+const CAMERA_FAR = 1000; // R3F's default
```

Comment:
It might make a tiny bit of an improvement to hard-code this value... Either way, it simplifies the code.